src/ORM/Table.php

```diff
@@ -1478,9 +1478,10 @@ public function findList(Query $query, array $options)
      */
     public function findThreaded(Query $query, array $options)
     {
+        $treeBehavior = $this->behaviors()->get('Tree');
```

Comment:
I'm not sure this method rely on `Tree behavior`

Reply:
You might have to use `has()` instead. Something like

```php
$parent = 'parent_id';
if ($this->behaviors()->has('Tree')) {
  $parent = $this->behaviors->get('Tree')->getConfig('parent');
}
```

Reply:
Actually, table class itself has method `hasBehavior()` :)